lms/djangoapps/notes/tests.py

```diff
@@ -39,11 +42,53 @@ def test_notes_enabled(self):
         Tests that notes are enabled when the course tab configuration contains
         a tab with type "notes."
         '''
-        self.course.tabs = [{'type': 'foo'},
-                            {'name': 'My Notes', 'type': 'notes'},
-                            {'type': 'bar'}]
+        with self.settings(FEATURES={'ENABLE_STUDENT_NOTES': True}):
```

Comment:
Hmmm? This isn't using `is_enabled`. This is a test that's overriding the django settings with its own feature flags.